src/AppBundle/Controller/DefaultController.php

```diff
@@ -4,15 +4,17 @@
 
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-use Symfony\Component\HttpFoundation\Request;
```

Comment:
We decided at some point to keep such a use statement to make it easier for people to start working right away. Not sure if we need to keep it now.